ce/test/tsconfig.json

```diff
@@ -1,14 +1,12 @@
 {
   "extends": "../common/tsconfig.json",
   "compilerOptions": {
-    "experimentalDecorators": true,
     "outDir": "./dist",
     "rootDir": ".",
     "types": [
       "node",
       "mocha"
-    ],
-    "inlineSourceMap": true,
```

Comment:
Was `inlineSourceMap` causing a problem?

Reply:
> Was `inlineSourceMap` causing a problem?

No, but it's already specified in the common one here:

https://github.com/microsoft/vcpkg-tool/blob/88386bbabd76e9a0418df0b23f0ba3e6ff1d9daf/ce/common/tsconfig.json#L11

(I deduplicated any such settings I noticed)